benchbuild/projects/polybench/polybench.py

```diff
@@ -82,7 +75,14 @@
         "floyd-warshall": "medley",
     }
 
-    SRC_FILE = "polybench.tar.gz"
+    SOURCE = [
+        HTTP(
+            remote={
+                '4.2':
+                    'http://downloads.sourceforge.net/project/polybench/polybench-c-4.2.tar.gz'  # pylint: disable=line-too-long
```

Comment:
Codacy found an issue: [Useless suppression of 'line-too-long'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)